py/private/py_pytest_main.bzl

```diff
@@ -53,13 +53,15 @@ _py_pytest_main = rule(
     },
 )
 
-def py_pytest_main(name, py_library = default_py_library, deps = [], **kwargs):
+def py_pytest_main(name, py_library = default_py_library, deps = [], data = [], testonly = False, **kwargs):
```

Comment:
Wouldn't `testonly=True` be a better default? Is there a use case for using the result from production code?

Reply:
I was debating about that. My rationale was that `testonly = False` is the `py_library` default, but didn't have strong reason.